pylint/config.py

```diff
@@ -537,6 +538,7 @@ def __init__(self):
         super(IniFileParser, self).__init__()
         self._parser = configparser.ConfigParser(
             inline_comment_prefixes=('#', ';'),
+            default_section='MASTER',
```

Comment:
Default section is customisable, so don't need to reimplent configparser.ConfigParser for that.